deb-packaging/src/deb/control/postrm

```diff
@@ -15,4 +15,6 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-/usr/sbin/userdel brooklyn
+if /usr/bin/getent passwd brooklyn; then
+    /usr/sbin/userdel brooklyn
```

Comment:
@johnmccabe If you think we shouldn't delete the user, I will delete the whole postrm file then.